tool.go

```diff
@@ -319,11 +319,13 @@ func main() {
 					}
 
 					for _, decl := range testPkg.Archive.Declarations {
-						if strings.HasPrefix(decl.FullName, testPkg.ImportPath+".Test") {
+						switch {
+						case decl.FullName == testPkg.ImportPath+".TestMain":
```

Comment:
This only checks that the func name matches "TestMain", but doesn't check that the func signature takes `m *testing.M` parameters and has no return variables, does it?

That potentially means that some Go package that has a `func TestMain(t *testing.T)` will get interpreted differently by `go` and `gopherjs`, if I understand this correctly.